### ISSUE TITLE
Provide some question examples

### ISSUE THREAD

**Issue by waldyrious** [Benestar/ask-wikidata#3]
For example, the home page could say something like:

> Type your question into the input box above and press enter. Or click one of the examples below:
> - When was Elvis born?
> - Where is the Eiffel tower?

And clicking the questions would automatically fill the search box and run the query.

**Comment by mettacrawler** [comment]
Too late, they broke "Where is the Eiffel Tower?" too.  The pattern "is located in" no longer matches any properties.

**Comment by Benestar** [comment]
Yes, that might be useful. However, you may want to try http://askplatyp.us/ which is far more advanced than this simple tool which is merely a proof of concept.
